test/spec/features/palette/PaletteSpec.js

```diff
@@ -798,6 +798,53 @@ describe('features/palette', function() {
     }));
   });
 
+
+  describe('event integration', function() {
+
+    const triggerSpy = sinon.spy();
+
+    const entry = {
+      label: 'My Entry',
+      action: () => {}
+    };
+
+    const paletteProvider = {
+      getPaletteEntries: function() {
+        return {
+          'entry-1': entry
+        };
+      }
+    };
+
+    beforeEach(bootstrapDiagram({ modules: [ paletteModule ] }));
+
+    beforeEach(inject(function(palette) {
+      palette.registerProvider(800, paletteProvider);
+    }));
+
+    afterEach(function() {
```

Comment:
I personally like to have _less_ stuff like this outside of a spec. Otherwise I have to context switch too much (scroll to see what "triggerSpy" is and so forth). `triggerSpy` could be setup inside the test (I think), and then we'd not need this.

Reply:
Gotcha, I moved it to inside the spec.